### PR TITLE
Bug 1877483 - add larch to all-nightly-branches

### PR DESCRIPTION
**Pull request by gbrownmozilla**

### CODE REVIEW

File: src/scriptworker/version.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def get_version_string(version: Union[ShortVerType, LongVerType]) -> str:
 
 # 1}}}
 # Semantic versioning 2.0.0  http://semver.org/
-__version__ = (50, 0, 0)
+__version__ = (50, 1, 0)
```
Comment (JohanLorenzo) [inline src/scriptworker/version.py:57]: Nit: Let's extract the version bump in a separate commit. If I'm not mistaken, it's the trend in this repo.

File: src/scriptworker/artifacts.py [hunk 2]
```diff
@@ -4,6 +4,7 @@
 in S3.
 
 """
+
```
Comment (JohanLorenzo) [inline src/scriptworker/artifacts.py:7]: Praise: Thanks for running the latest version of `black`!

**Comment by gbrownmozilla** [comment]
Bah, messed up the rebase. Let's start again!
